suplemon/logger.py

```diff
@@ -33,6 +33,8 @@ def close(self):
         self.acquire()
         try:
             for record in self.buffer:
+                if record.levelno < self.level:
```

Comment:
I'm pretty sure we don't need these lines. The default behavior should be `<=` since I would expect an "error" logger to log "error" level messages. Maybe our levels aren't properly configured for the loggers themselves?